src/state/endpoints/tests/reducer.test.js

```diff
@@ -0,0 +1,79 @@
+import deepfreeze from 'deepfreeze';
```

Comment:
Actually, you have to choose between having .test.js in the name or putting the files in a __tests___ directory. I did not like the `__tests__`, that's why I chose tests/*.test.js . What would you have done ? I can change that.